src/graph_notebook/magics/graph_magic.py

```diff
@@ -230,22 +228,18 @@ def sparql(self, line='', cell='', local_ns: dict = None):
             results = query_res.json()
             store_to_ns(args.store_to, results, local_ns)
 
-            table_output = widgets.Output(layout=DEFAULT_LAYOUT)
+            # table_output = widgets.Output(layout=DEFAULT_LAYOUT)
```

Comment:
Remove commented out line; I'm assuming this is no longer useful now that we are using first_tab_output